aws-iot-accountauditconfiguration/src/test/java/com/amazonaws/iot/accountauditconfiguration/TranslatorTest.java

```diff
@@ -69,6 +77,6 @@ void translateNotificationsFromIotToCfn_NonEmpty() {
 
     @Test
     void translateNotificationsFromIotToCfn_NullIn_EmptyOut() {
-        assertThat(Translator.translateNotificationsFromIotToCfn(null)).isEmpty();
+        assertThat(Translator.translateNotificationsFromIotToCfn(null)).isNull();
```

Comment:
Let's rename the test now that the assertion changed

Reply:
Done